### PR TITLE
Fix/revert basic party modal 

### PR DESCRIPTION
**Pull request by aivonix**
https://trello.com/c/QoSq0lTJ/8-make-basic-party-map-modals-become-the-full-black-popup-as-before

Fix to make a onClick from map to show the same modal as room-list, whilst keeping IS_BURN logic the same.

### CODE REVIEW

File: src/components/templates/Camp/components/Map.tsx [hunk 1]
```diff
@@ -64,18 +69,14 @@ export const Map: React.FC<PropsType> = ({
...
-                  setIsRoomModalOpen(true);
+                if (!IS_BURN) {
+                  openModal(room);
                 } else {
                   setRoomClicked((prevRoomClicked) =>
                     prevRoomClicked === room.title ? undefined : room.title
                   );
                 }
               }}
-              onMouseEnter={() => {
```
Comment (thecadams) [inline src/components/templates/Camp/components/Map.tsx:76]: You still need this for burn mode, you able to keep it?

File: src/components/templates/Camp/components/Map.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { CampRoomData } from "types/CampRoomData";
 import "./Map.scss";
 import { enterRoom } from "../../../../utils/useLocationUpdateEffect";
 import { useUser } from "../../../../hooks/useUser";
+import RoomCard from "./RoomCard";
```
Comment (thecadams) [inline src/components/templates/Camp/components/Map.tsx:7]: Can’t see where this is used, did the build succeed?

File: src/components/templates/Camp/components/Map.tsx (hunk 1 above)
Comment (aivonix) [reply src/components/templates/Camp/components/Map.tsx]: reverted that back in

File: src/components/templates/Camp/components/Map.tsx (hunk 2 above)
Comment (aivonix) [reply src/components/templates/Camp/components/Map.tsx]: removed this line

**Review by thecadams — APPROVED** [review]

File: src/components/templates/Camp/components/RoomModalOngoingEvent.tsx [hunk 3]
```diff
@@ -42,18 +42,49 @@ export const RoomModalOngoingEvent: React.FunctionComponent<PropsType> = ({
...
       )}
+      {eventToDisplay && !currentEvent && (
+        <>
+          <div className="title-container">
+            <img
+              src="/sparkle-icon.png"
+              className="sparkle-icon"
+              alt="sparkle-icon"
+            />
+            What's on next
```
Comment (thecadams) [inline src/components/templates/Camp/components/RoomModalOngoingEvent.tsx:55]: Can you please clean up the `whatsOnText` while in here?

More broadly let's keep an eye out for refactors leaving things in a half finished state. Cheers!

**Review by thecadams — CHANGES_REQUESTED** [review]
Looking good but there's some eye for detail type feedback that I'm hoping we can make sure we incorporate. Thanks for this!

**Review by thecadams — APPROVED** [review]
Looking good, thanks for taking care of all the feedback.

**Pull request merged**